lib/rails/generators/alchemy/base.rb

```diff
@@ -3,7 +3,7 @@
 module Alchemy
   module Generators
     class Base < ::Rails::Generators::Base
-      class_option :template_engine, :type => :string, :aliases => '-e', :desc => 'Template engine for the views. Available options are "erb", "haml", and "slim".'
+      class_option :template_engine, type: :string, aliases: '-e', desc: 'Template engine for the views. Available options are "erb", "haml", and "slim".'
```

Comment:
Metrics/LineLength: Line is too long. [154/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)